marketplace/core/types/channels/whatsapp_cloud/requests.py

```diff
@@ -0,0 +1,139 @@
+import json
+
+import requests
+from rest_framework import status
+
+from django.conf import settings
+from ..whatsapp_base.interfaces import ProfileHandlerInterface
+from ..whatsapp_base.exceptions import FacebookApiException
+
+
+class CloudProfileRequest(ProfileHandlerInterface):
+    # TODO: Validate response status
+    _endpoint = "/whatsapp_business_profile"
+    _fields = dict(fields="about,address,description,email,profile_picture_url,websites,vertical")
+
+    def __init__(self, phone_number_id: "str") -> None:
+        self._phone_number_id = phone_number_id
+
+    @property
+    def _url(self) -> str:
+        return settings.WHATSAPP_API_URL + f"/{self._phone_number_id}" + self._endpoint
+
+    @property
+    def _headers(self) -> dict:
+        access_token = settings.WHATSAPP_SYSTEM_USER_ACCESS_TOKEN
+        return {"Content-Type": "application/json", "Authorization": f"Bearer {access_token}"}
+
+    def get_profile(self):
+        response = requests.get(self._url, params=self._fields, headers=self._headers)
+        content = response.json().get("data", [])[0]
```

Comment:
I think you can change:
`content = response.json().get("data", [])[0]`
to:
`content = response.json().get("data", [{}])[0]`
Its prevents a IndexError